distributed/tests/test_ssl.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import print_function, division, absolute_import
+
+import subprocess
+
+from distributed import (Client, Scheduler, Worker)
+from distributed.utils_test import gen_cluster, inc, loop
+
+import pytest
+import os
+
+@pytest.fixture(scope="function")
+def ssl_kwargs(tmpdir):
+
+    ssl_dir = tmpdir.mkdir("ssl")
+    dirname = ssl_dir.dirname
+
+    keyfile = os.path.join(dirname, 'test.key')
+    certfile = os.path.join(dirname, 'test.pem')
+
+    p = subprocess.Popen([
+        'openssl', 'req',
```

Comment:
I thought about generating them using python, but that started looking painful pretty quickly.  Pregenerating them sounds like a decent idea.  Guess we should also explicitly tell people to NOT use them for anything other than test purposes.